src-cpp/app/modules/main/wallet/accounts/controller.cpp

```diff
@@ -22,4 +22,45 @@ QList<Wallets::WalletAccountDto> Controller::getWalletAccounts()
 
     return wallet_accounts;
 }
+
+QString Controller::generateNewAccount(QString password, QString accountName, QString color)
+{
+    if(m_walletServicePtr != nullptr)
```

Comment:
`minor` no need to check it like `== or !=` just `if (m_walletServicePtr) or (!m_walletServicePtr)`